packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/__tests__/barrier-selector.spec.js

```diff
@@ -75,8 +75,15 @@ describe('<BarrierSelector/>', () => {
     });
     it('barrier list should not be rendered when cross icon is clicked', () => {
         userEvent.click(current_barrier);
-        const icon_cross = screen.getByText('IcCross');
-        userEvent.click(icon_cross);
+        const icon_cross = screen.getAllByText('IcCross');
+        userEvent.click(icon_cross[0]);
+
+        expect(screen.queryByText(barriers_list_header)).not.toBeInTheDocument();
+    });
+    it('barrier list should not be rendered when the new barrier option was clicked', () => {
+        userEvent.click(current_barrier);
+        const clicked_barrier = screen.getByTestId(mock_props.turbos_barrier_choices[1]);
+        userEvent.click(cliked_barrier);
```

Comment:
another one

Reply:
Done! Also find 2 misspellings and correct them too